### PR TITLE
Add Explaination of which devices can boot from USB

### PR DESCRIPTION
**Pull request by Holyprogrammer**

### CODE REVIEW

**Comment by JamesH65** [comment]
I don't think that syntax will be understood by most, better to explicitly say 3A and 3B, 3A+ and 3B+.

File: hardware/raspberrypi/bootmodes/msd.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This tutorial explains how to boot your Raspberry Pi from a USB mass storage dev
 
 ## Program USB boot mode
 
-The Raspberry Pi 3+ is able to boot from USB without any changes, but the Raspberry Pi 3 requires the USB boot bit to be set in the OTP (one-time programmable) memory. If you are using a Raspberry Pi 3+, please go to the next section.
+The Raspberry Pi 3*+ is able to boot from USB without any changes, but the Raspberry Pi 3* requires the USB boot bit to be set in the OTP (one-time programmable) memory. If you are using a Raspberry Pi 3*+, please go to the next section.
```
Comment (andrum99) [inline hardware/raspberrypi/bootmodes/msd.md]: Change 3*+ to '3A+ and 3B+'

Change 3* to '3A and 3B'

as James says.

Comment (Holyprogrammer) [reply hardware/raspberrypi/bootmodes/msd.md]: Ok

**Comment by truongvan** [comment]
Can a Raspberry Pi 4 boot from USB without any change?

**Comment by lurch** [comment]
USB-boot hasn't been added to the Pi 4 firmware yet, but it will be added in the future.
https://www.raspberrypi.org/documentation/hardware/raspberrypi/booteeprom.md

**Pull request merged**